packages/core/src/styles/layout/_type-grid.scss

```diff
@@ -5,13 +5,11 @@
 @mixin generateColumnsAndRows($breakpoint: null) {
   @for $i from 1 through $cds-token-layout-grid-cols-static {
     // implicit columns/rows
-    [#{$layout}*='#{$cols}#{$breakpoint}:#{$i}'] > *,
-    [#{$layout}*='#{$cols}#{$breakpoint}:#{$i}']::slotted(*) {
+    [#{$layout}*='#{$cols}#{$breakpoint}:#{$i}'] > * {
       grid-column: span $i / span $i;
     }
 
-    [#{$layout}*='#{$rows}#{$breakpoint}:#{$i}'] > *,
-    [#{$layout}*='#{$rows}#{$breakpoint}:#{$i}']::slotted(*) {
```

Comment:
Removed internal slot support for grid since we dont use them. Likely in the future will want to avoid viewport based media queries internally in favor of container queries with ResizeObserver.